packages/nimble-blazor/NimbleBlazor.Acceptance.Tests/Pages/_Layout.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@using Microsoft.AspNetCore.Components.Web
+@namespace NimbleBlazor.Tests.Acceptance.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <base href="~/" />
```

Comment:
what does this do?.. This would be some asp.net specific behavior right?

Reply:
This is needed for relative paths to get resolved correctly by the server.
The MS docs about it are here: https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/?view=aspnetcore-7.0&tabs=visual-studio#app-base-path

I think our existing example apps get away with omitting it because they have a single page (served at `/`). In this test app if I remove it, resources fail to load. For example, `http://localhost:5202/DialogOpenAndClose` tries to load `http://localhost:5202/DialogOpenAndClose/_content/NimbleBlazor/NimbleBlazor.lib.module.js` and fails.

Reply:
(From the docs) Specifically the "~/" form means "the relative URL path to the app is the root of the app"